doc/pages/faq.asciidoc

```diff
@@ -59,14 +63,110 @@ $ tic /path/to/kakoune/contrib/tmux-256color.terminfo
 
 Finally, quit all existing sessions (`tmux kill-server`), and restart `tmux`.
 
-== Why does leaving insert mode take more than half a second in `tmux` ?
+=== Why do I see weird special characters (~T~@~U) around Clippy?
+
+You need a UTF-8 compliant locale, in order to render special characters.
+
+The way to do so might vary from distribution to another, but an easy way
+to verify that your locale is set correctly is to run the following command:
+
+----
+$ locale
+----
+
+The output should look similar to the following (with eventually another
```

Comment:
I think "eventually" is a false friend here, maybe: 

```
with your language and country code instead of `en_US`
```

Reply:
Thanks for the review, I've addressed the comment in a "prose fix" commit.